target/linux/ramips/dts/mt7628an_wiznet_wizfi630s.dts

```diff
@@ -61,13 +61,13 @@
 
 		wps {
 			label = "wps";
-			gpios = <&gpio 32 GPIO_ACTIVE_HIGH>;
+			gpios = <&gpio 41 GPIO_ACTIVE_HIGH>;
 			linux,code = <KEY_WPS_BUTTON>;
 		};
 
 		scm1 {
 			label = "SCM1";
-			gpios = <&gpio 5 GPIO_ACTIVE_LOW>;
+			gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
```

Comment:
If that's a switch, we should add
`linux,input-type = <EV_SW>;`
?

Reply:
Yes, two static sliders. I'll add it.